apps/projects/app/App.js

```diff
@@ -529,13 +525,6 @@ class App extends React.PureComponent {
 
     const navigationItems = [ 'Projects', ...(issueDetail ? ['Issue Detail'] : []) ]
 
-    // return (
-    //   <Main assetsUrl={ASSETS_URL}>
-
-
-    //   </Main>
-    // )
-
     console.log(this.props)
```

Comment:
Thanks for this cleanup!! it is really a best practice, could we remove the console.log as well? or put it on an special `log` function, only enabled on dev environment, as shown on profiles app :)

Reply:
Debugging TBD, but I'll be cleaning up as I go :)